inference-engine/tests/functional/plugin/cpu/bfloat16/memory_conv.cpp

```diff
@@ -0,0 +1,203 @@
+                </port>
+            </input>
+            <output>
+                <port id="2">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </output>
+        </layer>
+        <layer id="3" name="Activation_3_2" precision="FP32" type="Activation">
+            <data type="sigmoid" />
+            <input>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </input>
+            <output>
+                <port id="1">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </output>
+        </layer>
+        <layer id="4" name="Fc_4" precision="FP32" type="InnerProduct" >
+            <data out-size="200" />
+            <input>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </input>
+            <output>
+                <port id="1">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </output>
+            <weights offset="0" size="160000" />
+            <biases offset="160000" size="800" />
+        </layer>
+        <layer id="5" name="Memory_5" precision="FP32" type="Memory">
+            <data id="r_1-3" index="0" size="2" />
+            <input>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </input>
+        </layer>
+       <layer id="6" name="Fc_6" precision="FP32" type="InnerProduct"  >
+            <data out-size="200" />
+            <input>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </input>
+            <output>
+                <port id="1">
+                    <dim>1</dim>
+                    <dim>200</dim>
+                </port>
+            </output>
+            <weights offset="160800" size="160000" />
+            <biases offset="320800" size="800" />
+        </layer>
+    </layers>
+    <edges>
+        <edge from-layer="0" from-port="0" to-layer="2" to-port="0" />
+        <edge from-layer="1" from-port="0" to-layer="2" to-port="1" />
+        <edge from-layer="2" from-port="2" to-layer="3" to-port="0" />
+        <edge from-layer="3" from-port="1" to-layer="4" to-port="0" />
+        <edge from-layer="4" from-port="1" to-layer="5" to-port="0" />
+        <edge from-layer="4" from-port="1" to-layer="6" to-port="0" />
+    </edges>
+</net>
+)V0G0N";
+    InferenceEngine::Blob::Ptr net_bin_v6;
+};
+
+TEST_P(MemoryConv, inferV6Model) {
+    if (!InferenceEngine::with_cpu_x86_bfloat16()) {
+        // on platforms which do not support bfloat16, we are disabling bf16 tests since there are no bf16 primitives,
+        // tests are useless on such platforms
+        return;
```

Comment:
please use GTEST_SKIP()

Reply:
The request is not for this PR. This approach is already commonly used for BF16 tests. I don't like this approach as mach as you. But it should be redesigned for all BF16 test cases.